promdump/promdump.go

```diff
@@ -165,6 +173,7 @@ func writeFile(values *[]*model.SampleStream, filePrefix string, fileNum uint) e
 		log.Printf("writeFile: writing %v results to file %v", len(*values), filename)
 	}
 	return os.WriteFile(filename, valuesJSON, 0644)
+
```

Comment:
Spurious whitespace change. No need to worry about it this time but try to avoid in future.